src/app/organisms/room/RoomViewHeader.jsx

```diff
@@ -40,10 +40,8 @@ function RoomViewHeader({ roomId }) {
   const isDM = initMatrix.roomList.directs.has(roomId);
   const room = mx.getRoom(roomId);
   const setPeopleDrawer = useSetSetting(settingsAtom, 'isPeopleDrawer');
-  let avatarSrc = room.getAvatarUrl(mx.baseUrl, 36, 36, 'crop');
-  avatarSrc = isDM
-    ? room.getAvatarFallbackMember()?.getAvatarUrl(mx.baseUrl, 36, 36, 'crop')
-    : avatarSrc;
+  let avatarSrc = room.getAvatarFallbackMember()?.getAvatarUrl(mx.baseUrl, 24, 24, 'crop') || null;
+  if (avatarSrc === null) avatarSrc = room.getAvatarUrl(mx.baseUrl, 36, 36, 'crop') || null;
```

Comment:
while it fixed it for the dms it broke it for the home/space rooms as now they will how fallback member avatar if member count is 2